code/game/objects/items/rogueweapons/melee/swords.dm

```diff
@@ -82,6 +82,22 @@
 	swingdelay = 0
 	item_d_type = "stab"
 
+/obj/item/rogueweapon/sword/stone
+	force = 15 //Weaker than a short sword
+	force_wielded = 20 // Less than other swords
+	possible_item_intents = list(/datum/intent/sword/cut, /datum/intent/sword/thrust)
+	gripped_intents = list(/datum/intent/sword/cut, /datum/intent/sword/thrust, /datum/intent/sword/chop)
+	name = "stone sword"
+	desc = "A simple stone sword, crude and effective."
+	icon_state = "stone_sword"
+	max_blade_int = 100
+	max_integrity = 125
```

Comment:
this also gives it functionally the same durability as iron swords. the stone knife has 50 in each by comparison.